readme.md

```diff
@@ -104,6 +104,15 @@ Sets `options.method` to the method name and makes a request.
 
 Type: `Object`
 
+##### method
+
+Type: `String`
+Default: `'GET'`
```

Comment:
Since we normalize anyway, we can use `get` here to show users clearly that they can use lowercase.